README.md

```diff
@@ -77,26 +100,6 @@ sudo cp $GOPATH/bin/eigenlayer /usr/local/bin/
 sudo cp eigenlayer/build/eigenlayer /usr/local/bin/
 ```
 
-## Install `eigenlayer` CLI using a binary
-
-The `eigenlayer` CLI tool versions are managed with GitHub releases. To install it, you can download the binary directly from the release assets manually, or by using the following command replacing the `<VERSION>` and `<ARCH>` with the proper values:
-
-```bash
-curl -L https://github.com/NethermindEth/eigenlayer/releases/download/<VERSION>/eigenlayer-linux-<ARCH> --output eigenlayer
-```
-
-### Linux/amd64
-
-```bash
-curl -L https://github.com/NethermindEth/eigenlayer/releases/download/v0.4.0/eigenlayer-linux-amd64 --output eigenlayer
-```
-
-### Linux/arm64
-
-```bash
-curl -L https://github.com/NethermindEth/eigenlayer/releases/download/v0.4.0/eigenlayer-linux-arm64 --output eigenlayer
-```
-
 ## Create and List Keys
```

Comment:
Can we change this header to "Create or Import" keys, since those are the two most important actions for the user at this point?

Can we add a sentence at the top of this section that explains something like the following:

> ECDSA and BLS keys are required for operator registration. You may choose to either create your own set of keys using the EigenLayer CLI and local keystore (recommended for first time users) or import keys from your remote keystore (recommended for advanced users that maintain a remote keystore service).

Reply:
oh this is nice. I will add these in a new PR.